package.json

```diff
@@ -95,7 +95,7 @@
     "vue-cli-plugin-vuetify": "^2.5.8",
     "vue-plugin-helper-decorator": "^0.0.12",
     "vue-property-decorator": "^9.1.2",
-    "vue-template-compiler": "2.7.14",
+    "vue-template-compiler": "2.7.10",
```

Comment:
I have the version 2.7.14 in my Filings UI package updates ticket. Should I also downgrade there?

Reply:
Yes.